guides/v2.2/cloud/project/project-conf-files_services-rabbit.md

```diff
@@ -108,6 +108,7 @@ For debugging purposes, it is useful to directly connect to a service instance i
    ```
 
 1. While the session is open, you can start a RabbitMQ client of your choice from your local workstation, configured to connect to the `localhost:<portnumber>` using the port number, username, and password information from the MAGENTO_CLOUD_RELATIONSHIP variable.
+	 The RabbitMQ management plugin is installed in cloud, so there is a graphical utility that can be used for debugging purposes and it is usually available at port 15672.
```

Comment:
Hi, 
if usually available at port 15672, but for some reason you're not in this port, how would you know which port you're in? the developer should open a ticket?

Reply:
Maybe @hguthrie can talk with Magento Cloud team to clarify it. And you can do the changes addressed from Cloud team.

Reply:
I am researching this. Thanks for the notice! I'll get back to you soon.